docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1,197 +1,107 @@
-   * [redux](https://www.npmjs.com/package/redux)
-   * [react-router](https://github.com/reactjs/react-router)
-   * [react-router-redux](https://github.com/reactjs/react-router-redux)
-   * [react-redux](https://www.npmjs.com/package/react-redux)
-   * [react-dom](https://www.npmjs.com/package/react-dom)
-   * [isomorphic-fetch](https://www.npmjs.com/package/isomorphic-fetch)
-   * [redux-thunk](https://www.npmjs.com/package/redux-thunk)
-   * [react-addons-test-utils](https://www.npmjs.com/package/react-addons-test-utils)
- * Added SilverStripeComponent base class for building React UIs
-
-### Compiling CSS
-
-[Compass](http://compass-style.org/) has been removed and css/sprite compilation is now handled with [gulp](http://gulpjs.com/) (see [Markup and Style Conventions](../developer_guides/customising_the_admin_interface/cms_architecture/#markup-and-style-conventions)).
-
-### Bootstrap CSS
-
-Bootstrap 4 Alpha.2 CSS is available in `admin/css/bootstrap`.
-
-### Changed paths of static assets
+## Introduction
+
+This version introduces many breaking changes, which in most projects can be managed through a combination
+of automatic upgrade processes as well as manual code review. This document reviews these changes and will
+guide developers in preparing existing 3.x code for compatibility with 4.0
+
+* [Highlights of Major Changes](#overview)
+* [Upgrading Guide](#upgrading)
+  * [Standard Upgrade](#upgrading-primary)
+  * [API Specific Upgrades](#upgrading-specifics)
+* [API Changes](#api-changes)
+  * [General and Core API](#overview-general)
+  * [ORM API](#overview-orm)
+  * [Filesystem API](#overview-filesystem)
+  * [Template and Form API](#overview-template)
+* [Commit History](#commit-history)
+
+## <a name="overview"></a>Highlights of major changes
+
+* Minimum version dependencies have increased; PHP 5.5 and Internet Explorer 10 (or other modern browser)
+  is required.
+* All code earlier marked as deprecated for 4.0 has now been removed.
+* All SilverStripe classes are now namespaced, and some have been renamed. This has major implications for
+  arrangement of templates, as well as other references to classes via string literals or configuration.
+  Automatic upgrading tools have been developed to cope with the bulk of these changes (see upgrading notes).
+* Asset storage has been abstracted, and a new concept of `DBFile` references via database column references
+  now exists in addition to references via the existing `File` dataobject. File security and protected files
+  are now a core feature.
+* A new front-end development process has been developed for the construction of javascript based components,
+  prominently featuring ReactJS to develop highly functional CMS content areas. A new standard form schema
+  API has been developed to allow back-end PHP constructed forms to scaffold themselves within ReactJS
+  powered sections.
+* CMS CSS has been re-developed using bootstrap 4 as a base.
```

Comment:
Will link to https://www.silverstripe.org/blog/a-frameworks-framework-why-silverstripe-4-will-use-bootstrap/